### PR TITLE
Fix Typo

### PR DESCRIPTION
**Pull request by mijin85cho**
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

### Change Description ###

Some typos are reported. So I raise this PR to fix the typos.

### CODE REVIEW

**Review by annie-samsung — APPROVED** [review]
Changes look fine, hence approving

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ We are going to review <span class="labels lh-default d-block d-md-inline">
 3. When one of reviewers approve a PR, he/she has to update the label to <span class="labels lh-default d-block d-md-inline">
     <a class="d-inline-block IssueLabel" style="background-color: #0523aa; color: #ffffff" title="Ready for publishing" href="https://github.com/Samsung/tizen-docs/pulls?q=is%3Apr+is%3Aopen+label%3AReview%2FScheduled">Review/Scheduled</a>
     </span>
-4. Althought the state is <span class="labels lh-default d-block d-md-inline">
+4. Although the state is <span class="labels lh-default d-block d-md-inline">
```
Comment (annie-samsung) [inline README.md:34]: one minor comment:

I think we can remove the comma and the
, the others can change >>>  others can change

Thoughts?

Comment (mijin85cho) [reply README.md:34]: Updated. Would you please check again?

**Review by annie-samsung — APPROVED** [review]
Approving, changes look fine.

**Pull request merged**